packages/grafana-data/src/types/plugin.ts

```diff
@@ -43,6 +43,8 @@ export enum PluginErrorCode {
   missingSignature = 'signatureMissing',
   invalidSignature = 'signatureInvalid',
   modifiedSignature = 'signatureModified',
+  failedStart = 'failedStart',
```

Comment:
```suggestion
  failedBackendStart = 'failedBackendStart',
```

This and similar suggestions are entirely optional and I'd defer to your judgement, but I'm wondering if being more explicit here generally makes it clearer